pkgs/development/libraries/gtkextra/default.nix

```
@@ -0,0 +1,49 @@
+{ stdenv,
+  lib,
+  fetchurl,
+  pkg-config,
+  gobject-introspection,
+  file,
+  gtk2,
+  glib,
+  cairo,
+  atk,
+  pango,
+  libtiff,
+  libpng,
+  libjpeg
+}:
+
+let
+  minorVersion = "3.3";
+  microVersion = "4";
+  version = "${minorVersion}.${microVersion}";
+in
+stdenv.mkDerivation rec {
+
+  pname = "gtkextra";
+
+  inherit version;
+
+  src = fetchurl {
+    url = "mirror://sourceforge/project/gtkextra/${minorVersion}/${pname}-${version}.tar.gz";
+    sha256 = "1mpihbyzhv3ymfim93l9xnxmzhwyqdba5xb4rdn5vggdg25766v5";
+  };
+
+  preConfigure = ''
```

```suggestion
  postPatch = ''
```

Does that still work in preConfigure or do we miss a phase?

---

yep it works